packages/now-cli/src/commands/remove.js

```diff
@@ -86,34 +86,36 @@ export default async function main(ctx) {
   const output = createOutput({ debug: debugEnabled });
   const { success, error, log } = output;
 
-  if (ids.length < 1) {
-    error(`${cmd('now rm')} expects at least one argument`);
+  if (argv.help || ids[0] === 'help') {
     help();
-    return 1;
+    return 2;
   }
 
-  if (argv.help || ids[0] === 'help') {
+  if (ids.length < 1) {
+    error(`${cmd('now rm')} expects at least one argument`);
     help();
-    return 2;
+    return 1;
   }
```

Comment:
The only change made is inverting the two "ifs" above 👆The rest is `prettier` formatting the file.